models/novels.js

```diff
@@ -0,0 +1,9 @@
+const novels = (connection, Sequelize) => {
```

Comment:
you forgot to reference Authors model in this line
and the appropriate line for your foreign key in your models folder would be this
authorId: { type: Sequelize.INTEGER, references: { model: Authors, key: 'id' } },